### PR TITLE
New SAM Init Starter App: Event Bridge Hello World for EC2 instance change notifications (python3.6/7/8)

### PR DESCRIPTION
**Pull request by nikp**
Same idea as https://github.com/awslabs/aws-sam-cli-app-templates/pull/3 but in Python.

If the ideas seem sound, let me know before merging - I'll clone for python3.7

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nikp** [comment]
tested:

sam init, build, local invoke, package, deploy, and pytest per readme for unit tests

File: python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* [VS Code](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/welcome.html)
+* [Visual Studio](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/welcome.html)
+
+## Deploy the sample application
+
+The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. It uses Docker to run your functions in an Amazon Linux environment that matches Lambda.
+
+To use the SAM CLI, you need the following tools.
+
+* AWS CLI - [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and [configure it with your AWS credentials].
```
Comment (awood45) [inline python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md]: We can deal with this after launch, but there's a new starting point README with the new deploy instructions.

File: python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+The SAM CLI installs dependencies defined in `hello_world/requirements.txt`, creates a deployment package, and saves it in the `.aws-sam/build` folder.
+
+Test a single function by invoking it directly with a test event. An event is a JSON document that represents the input that the function receives from the event source. Test events are included in the `events` folder in this project.
+
+Run functions locally and invoke them with the `sam local invoke` command.
+
+```bash
+{{ cookiecutter.project_name }}$ sam local invoke HelloWorldFunction --event events/event.json
```
Comment (awood45) [inline python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md]: Would this work in your example here?

Comment (nikp) [reply python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md]: Yes, I've tested sam local invoke in all cases, though usually i don't provide the funciton name - just `sam local invoke --event events/event.json`

File: python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md (hunk 1 above)
Comment (nikp) [reply python3.6/cookiecutter-aws-sam-eventBridge-python/{{cookiecutter.project_name}}/README.md]: Sriram asked for the same readme update - i'll update it after all in all repos to be based on the latest to minimize confusion

**Comment by nikp** [comment]
java equiavelent of changes/cloning: https://github.com/awslabs/aws-sam-cli-app-templates/pull/10

**Comment by aahung** [comment]
Note: the missing files are pulled by `sam init` automatically with AWS creds.

**Pull request closed without merging**